.github/workflows/Pipeline.yml

```diff
@@ -44,20 +45,99 @@ jobs:
       report: 'htmlmypy'
       artifact: ${{ fromJson(needs.Params.outputs.params).artifacts.typing }}
 
-  PublishTestResults:
-    uses: pyTooling/Actions/.github/workflows/PublishTestResults.yml@r0
-    needs:
-      - UnitTesting
-
   Package:
     uses: pyTooling/Actions/.github/workflows/Package.yml@r0
     needs:
       - Params
-      - Coverage
+#      - Coverage
     with:
       python_version: ${{ fromJson(needs.Params.outputs.params).python_version }}
       artifact: ${{ fromJson(needs.Params.outputs.params).artifacts.package }}
 
+  Install:
```

Comment:
In don't completely understand this new job. It seems that the purpose is to execute unit tests. Therefore, it is a variant of UniTesting, rather than a new job. In fact, exactly the same matrix is used, because Params is not modified. Therefore, I believe it should be implemented by calling UnitTesting twice, probably changing some input/option.

Reply:
See also https://github.com/pyTooling/Actions/issues/32. So, in order to test the installation "cleanly", we might want to use the testing PyPI instead of artifacts. That'd make the Install job significantly different from the UnitTesting.

Reply:
I don't think testing installations from PyPI is an option, because either PyPI or GitHub Actions Cache servers have huge delays. When publishing package A and then merging code for package B, I often see 20% of CI jobs failing because old package versions are reported from cache servers. We would need huge delays or retries, in the pipeline to compensate this behavior.

Reply:
Did you see https://github.com/mithro/actions/blob/main/includes/workflows/python/install-and-test/workflow.yaml ?

Reply:
Job is removed from current branch / PR.